### PR TITLE
add usal residents to new unit

### PR DESCRIPTION
**Pull request by JacobHarrisonONS**

### CODE REVIEW

File: src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/impl/NewAddressReportedProcessor.java [hunk 1]
```diff
@@ -66,6 +66,8 @@ public UUID process(OutcomeSuperSetDto outcome, UUID caseIdHolder, String type)
     root.put("eventDate", eventDateTime);
     root.put("addressType", type);
     root.put("region", regionLookup(outcome.getOfficerId()));
+    root.put("usualResidents",
+        outcome.getCeDetails().getUsualResidents() != null ? outcome.getCeDetails().getUsualResidents() : "0");
```
Comment (garethTurner) [inline src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/impl/NewAddressReportedProcessor.java:70]: shouldn't this be in the if statement above where you check if getCeDetails is null

File: src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/impl/NewUnitAddressLinkedProcessor.java [hunk 2]
```diff
@@ -65,6 +65,11 @@ public UUID process(OutcomeSuperSetDto outcome, UUID caseIdHolder, String type)
     root.put("eventDate", eventDateTime);
     root.put("addressType", type);
     root.put("addressLevel", "U");
+    if (type.equals("CE") && outcome.getCeDetails() != null && outcome.getCeDetails().getUsualResidents() != null) {
```
Comment (garethTurner) [inline src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/impl/NewUnitAddressLinkedProcessor.java:68]: can type be null

File: src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/impl/NewAddressReportedProcessor.java (hunk 1 above)
Comment (JacobHarrisonONS) [reply src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/impl/NewAddressReportedProcessor.java]: nice spot

File: src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/impl/NewUnitAddressLinkedProcessor.java (hunk 2 above)
Comment (JacobHarrisonONS) [reply src/main/java/uk/gov/ons/census/fwmt/outcomeservice/converter/impl/NewUnitAddressLinkedProcessor.java:68]: type is passed from the controller survey type, can never be null

**Review by garethTurner — APPROVED** [review]

**Pull request merged**